src/js/stores/MesosStore.js

```diff
@@ -0,0 +1,221 @@
+var EventEmitter = require("events").EventEmitter;
+
+var AppDispatcher = require("../AppDispatcher");
+var InfoStore = require("../stores/InfoStore");
+var InfoActions = require("../actions/InfoActions");
+var InfoEvents = require("../events/InfoEvents");
+var MesosActions = require("../actions/MesosActions");
+var MesosEvents = require("../events/MesosEvents");
+
+const FILES_TTL = 60000;
+const STATE_TTL = 60000;
+const MASTER_ID = "master";
+
+var stateMap = {};
+var taskFileMap = {};
+var taskFileRequestQue = [];
+
+var MesosStore = Object.assign({
+
+  getState: function (nodeId) {
+    var data = stateMap[nodeId];
+    if (data == null || data.timestamp + STATE_TTL < Date.now()) {
```

Comment:
Yes, please rewrite it to eliminate the confusion, otherwise I hadn't commented on that.
Thx!